posthog/api/person.py

```diff
@@ -208,6 +208,16 @@ def list(self, request: request.Request, *args: Any, **kwargs: Any) -> response.
 
         return Response({"results": serialized_actors, "next": next_url, "previous": previous_url})
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "delete_events",
+                OpenApiTypes.BOOL,
+                description="If true, a task to delete all events associated with this person will be created and queued. The task does not run immediately and instead is batched together and at 5AM UTC every Sunday",
```

Comment:
```suggestion
                description="If true, a task to delete all events associated with this person will be created and queued. The task does not run immediately and instead is batched together and at 5AM UTC every Sunday (controlled by environment variable CLEAR_CLICKHOUSE_REMOVED_DATA_SCHEDULE_CRON)",
```

They're all better than where we started but... 

By listing the current value of the cron setting we can change the value without updating the docs. There's nothing to stop this getting out of sync

I think it should reference the cron job so that someone can track it down.

Reply:
@lharries 

We're way into bike shedding though so 🤷